.github/workflows/ort.yml

```diff
@@ -215,4 +211,27 @@ jobs:
                 GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
                 EVENT_NAME: ${{ github.event_name }}
                 INPUT_VERSION: ${{ github.event.inputs.version }}
-  
+
+            - name: Get current date
+              id: date
+              run: |
+                CURR_DATE=$(date +'%Y-%m-%d-%H')
+                echo "date=${CURR_DATE}" >> $GITHUB_OUTPUT
+
+            - name: Upload the final package list
+              continue-on-error: true
+              uses: actions/upload-artifact@v4
+              with:
+                  name: final-package-list-${{ steps.date.outputs.date }}
+                  path: |
+                      utils/final_package_list.txt
+                  retention-days: 3
```

Comment:
Why 3?

Reply:
Fixed to 30. This workflow runs every day, and the default is 90 days. We don't need to store it for a longer time. It will only be used for distribution reviews or perhaps if we'll need to debug to see when a dependency were added removed.